node_modules/rimraf/package.json

```diff
@@ -1,27 +1,29 @@
 {
-  "_from": "rimraf@^2.2.8",
+  "_from": "rimraf",
```

Comment:
why did you post this file

Reply:
When i was going through babel issues yesterday, rimraf was added to build script for performing a unix command rm -rf on a folder, but it turns out to be an ineffective approach so i dumped, but i didn't know it was pushed, next i am pushing i will add node_modules to gitignore